README.md

```diff
@@ -42,6 +42,33 @@ module.exports = {
 };
 ```
 
+## Gts/Gjs eslint setup
```

Comment:
Can you add a few high-level sentences explaining why someone should bother with this Gts/Gjs setup? For someone who isn't familiar with Gts/Gjs, what is it? How can they learn more about it? Why should anyone care?

Reply:
do you mean like, links to the RFC, a brief blurb about Polaris, etc?

Reply:
Yes, a one-sentence overview and links to docs/guides helps.